src/PrestaShopBundle/Controller/Admin/LegacyController.php

```diff
@@ -230,4 +266,16 @@ protected function initController(Request $request, array $dispatcherHookParamet
 
         return $adminController;
     }
+
+    private function isMethodOverridden(AdminController $adminController, string $methodName): bool
+    {
+        try {
+            $reflector = new ReflectionMethod($adminController, 'getTabSlug');
+
+            return $reflector->getDeclaringClass()->getName() !== AdminControllerCore::class;
+        } catch (ReflectionException) {
+        }
```

Comment:
The method is alays present because it's implemented by the parent AdminController anyway, I just added this catch because I was afraid of the similar bug on symfony routes because I don't want this reflection part to break the controller 🤔